fcrepo-kernel-api/src/main/java/org/fcrepo/kernel/api/models/ExternalContent.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Licensed to DuraSpace under one or more contributor license agreements.
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * DuraSpace licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except in
+ * compliance with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.fcrepo.kernel.api.models;
+
+import java.io.InputStream;
+
+/**
+ * Interface for the ExternalContent information class.
+ * @author bseeger
+ */
+public interface ExternalContent {
+
+    String PROXY = "proxy";
```

Comment:
🤷‍♂ They were in there entirely own file before, but as they are part of ExternalContent I thought they could stay here. An enum might be a good idea to make checks more consistent? Not sure.

Reply:
Hmm, maybe its not worth the trouble unless we find ourselves using these in a lot of places. It would be kind of nice/safer to be able to pass an enum around instead of a string for the handling though